test/useForm.test.tsx

```diff
@@ -264,6 +264,110 @@ describe('useForm', () => {
 
     expect(result.current.validate()).toEqual(false)
   })
+
+  it('should reset field value to default', async () => {
+    const { result } = render<Widget>({
+      name: field(),
+      details: {
+        description: field(),
+        picture: field()
+      },
+      components: field()
+    })
+
+    // update field value
```

Comment:
We can drop comments like these across the tests since they aren't really telling us anything the code isn't.